src/platform/user/profile/vet360/components/AddressField/AddressEditModal.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
turned this into a connected component so that it can grab the modal data from the redux store.